libnavui-androidauto/build.gradle

```
@@ -43,9 +43,12 @@ dependencies {
     // build.gradle.
     def carNavVersion = "2.8.0-beta.3"
     def carSearchVersion = "1.0.0-beta.35"
-    api("com.mapbox.navigation:android:${carNavVersion}")
     api("com.mapbox.search:mapbox-search-android:${carSearchVersion}")
-    implementation("com.mapbox.navigation:ui-app:${carNavVersion}")
+
```

```suggestion
// TODO https://mapbox.atlassian.net/browse/NAVSDK-544
//    Remove after upgrading to 2.8.0-rc.1
```

Will you add a comment here because this is temporary

---

Hmm actually I take this back, having a debug api is great

---

There is some concern that, our debug builds are never really the release builds..